### PR TITLE
[tests-only][full-ci] implement test for sharee trying to lock shared file/folder in secure view in lockFiles feature file

### PR DESCRIPTION
**Pull request by anon-pradip**
## Description

This PR updates the previous scenarios regarding sharee trying to lock shared file/folder with permission secure view .
Scenario Updated:
```feature
Scenario: viewer cannot lock a file in the shares using file-id
```

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/9334

## How Has This Been Tested?

- test environment:
- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiLocks/lockFiles.feature [hunk 1]
```diff
@@ -214,18 +214,22 @@ Feature: lock files
       | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario: viewer cannot lock a file in the shares using file-id
+  Scenario Outline: viewer cannot lock a file in the shares using file-id
```
Comment (S-Panta) [inline tests/acceptance/features/apiLocks/lockFiles.feature:217]:
```suggestion
  Scenario Outline: resource shared with other role except editor cannot lock a file in the shares using file-id
```

Comment (anon-pradip) [reply tests/acceptance/features/apiLocks/lockFiles.feature:217]: I think keeping "viewer" in the scenario gives more clearance. May be we can make it like:
`viewer with permissions other than editor cannot lock a file in the shares using file-id`

Comment (saw-jan) [reply tests/acceptance/features/apiLocks/lockFiles.feature:217]: `Scenario Outline: viewer cannot lock a file in the shares using file-id` is also fine

**Review by saw-jan — APPROVED** [review]

**Review by S-Panta — APPROVED** [review]
LGTM

**Pull request merged**